docs/community/contribute.md

```diff
@@ -24,14 +24,14 @@ Following these guidelines shows that you respect the time of the developers who
 
 ### What Can I Contribute?
 
-We are a fully Open Source project, meaning we are open to all kinds of contributions from our community. Here are a few examples of contributions that we are open to in this repo:
+We are a fully open source project, meaning we are open to all kinds of contributions from our community. Here are a few examples of contributions that we are open to:
 
-- **Added Content:** Writing a nice guide? Submit it (Where?) so we can add it to our Docs. 
-- **Translation:** Adding Translations of our prior content? See the [translate](TRANSLATE) page for help.
-- **Bug Reports & Fixes:** Find a bug or error? Let us know where it's hiding. (Where?)
+- **Added Content:** Writing a nice guide? Submit it on [Github](https://github.com/subspace/subspace-docs) so we can add it to our Docs. 
```

Comment:
Can we ask to have it posted on the [guides section of the forum](https://forum.subspace.network/c/guides-and-faqs/13)?